components/shared/ui/appPage.tsx

```diff
@@ -5,31 +5,79 @@ import { perCollectionSEOTitle } from "../../../lib/constants/labels";
 import { ValidCollectionCodename } from "../../../lib/types/perCollection";
 import { useSmartLink } from "../../../lib/useSmartLink";
 import { createItemSmartLink } from "../../../lib/utils/smartLinkUtils";
-import { Block_Navigation } from "../../../models";
+import { Article, Block_Navigation, Product, SEOMetadata, WSL_WebSpotlightRoot } from "../../../models";
 import { SiteCodenameProvider } from "../siteCodenameContext";
 import { Footer } from "./footer";
 import { Menu } from "./menu";
 
 type Props = Readonly<{
   children: ReactNode;
   siteCodename: ValidCollectionCodename;
-  itemId?: string;
+  item?: WSL_WebSpotlightRoot | Article | Product;
   siteMenu?: Block_Navigation;
+  defaultMetadata?: SEOMetadata;
```

Comment:
Why is this optional? I think you always need `defaultMetadata` just in case the provided `item` doesn't have any filled in, right?

Reply:
set to required